package.json

```diff
@@ -45,6 +46,9 @@
     "ethers": "^5.6.2",
     "graphql": "^16.3.0",
     "graphql-request": "^4.2.0",
+    "imagemagick": "^0.1.3",
+    "ipfs-api": "^26.1.2",
+    "ipfs-http-client": "^56.0.3",
```

Comment:
i decided to use this just because its was easier for me, happy to use non sugar client, i see you mostly interact with axios